tracing-subscriber/src/fmt/format/mod.rs

```diff
@@ -530,7 +550,10 @@ impl<F, T> Format<F, T> {
 
     /// Enable ANSI terminal colors for formatted output.
     pub fn with_ansi(self, ansi: bool) -> Format<F, T> {
```

Comment:
we _may_ want to deprecate this method, since the `fmt::Subscriber::with_ansi` method is probably more useful in most cases.